dlrover/python/elastic_agent/torch/training.py

```diff
@@ -140,7 +141,7 @@ class ElasticLaunchConfig(LaunchConfig):
     network_check: bool = False
     comm_perf_test: bool = False
     node_unit: int = 1
-    training_port: int = 60000
+    training_port: int = 64000
```

Comment:
reuse the same constant after fixing the "moving" comments above.

Reply:
lgtm